inc/thorvg.h

```diff
@@ -705,7 +705,8 @@ class TVG_API LinearGradient final : public Fill
      * @param[in] x2 The horizontal coordinate of the second point used to determine the gradient bounds.
      * @param[in] y2 The vertical coordinate of the second point used to determine the gradient bounds.
      *
-     * @note In case the first and the second points are equal, an object filled with such a gradient fill is not rendered.
+     * @note In case the first and the second points are equal, an object is filled with a single color using the last color specified in the colorStops().
```

Comment:
This hurts the performance. Maybe we can consider to switch using solid fill that case.

Reply:
I saw the comment, I'll handle this asap, but I don;t think this will affect perf so often, this is a very rare case

Reply:
@hermet please take a look in the tvgSwFill. this is already optimized, for both, radial and linear grad. 
radial was drawn before this change, but with a wrong color, linear was sometimes drawn sometimes not. if it was drawn, then the color was wrong